application/__init__.py

```
@@ -25,12 +25,22 @@
+from datetime import datetime
 
 logger = logging.getLogger(__name__)
 
 try:
     with open('log_config.yaml') as config_file:
         config = yaml.safe_load(config_file.read())
+    if 'handlers' in config and 'CLP_file' in config['handlers']:
+        logDir = "logs"
+        os.makedirs(logDir, exist_ok=True)
+
+        currentDateTime = datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)